pkg/apis/monitoring/v1/types.go

```diff
@@ -489,6 +489,13 @@ type PrometheusSpec struct {
 	// Note that if TargetLimit is lower, that value will be taken instead,
 	// except if either value is zero, in which case the non-zero value will be
 	// used.  If both values are zero, no limit is enforced.
+
+	// Exemplars related settings that are runtime reloadable.
+	Exemplars *Exemplars `json:"exemplars,omitempty"`
+}
+
+type Exemplars struct {
+	MaxSize uint `json:"maxSize,omitempty"`
```

Comment:
Prometheus uses `int64`. We need to make it a pointer to distinguish between "use Prometheus default value" (e.g. 100000) and "disable exemplar storage" (e.g. 0).

```suggestion
	// Maximum number of exemplars stored in memory for all series.
	// If not set, Prometheus uses its default value.
	// A value of zero or less than zero disables the storage.
	MaxSize *int64 `json:"maxSize,omitempty"`
```